### PR TITLE
Go - Move to Hold

### PR DESCRIPTION
**Pull request by piers-williams**
- [x] Are you happy for the content of this change to be publicly visible?
- [x] Are all new or updated entries marked as `isNew` = `true`?
- [x] Does `radar.csv` render correctly when viewed on Github? [Here](https://github.com/red-gate/Tech-Radar/blob/9e68431467c6a77ee6069849d96df28e4b1494ee/radar.csv)
- [x] Does the updated [tech radar](https://radar.thoughtworks.com/?documentId=https%3A%2F%2Fraw.githubusercontent.com%2Fred-gate%2FTech-Radar%2Fgo-hold%2Fradar.csv) display as you expect?

### CODE REVIEW

**Comment by clivetong** [comment]
> But it is now possible to achieve good results in C# anyway

C# 9 has a focus on cloud nativity - see https://devblogs.microsoft.com/dotnet/our-vision-for-dotnet-9/#platform-for-cloud-native-developers including AOT/DATAS

ps you need a full stop at the end of your added sentence

File: radar.csv [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name,ring,quadrant,isNew,description
 "C#",adopt,"Languages and Frameworks",true,"<a href='https://docs.microsoft.com/en-us/dotnet/csharp/'>C#</a> is our language of choice. It's strongly-typed, blends functional and object-oriented programming and works on all the OS we need to support."
 "TypeScript",adopt,"Languages and Frameworks",true,"<a href='https://www.typescriptlang.org/'>TypeScript</a> is our default choice for running code in the browser. It's strongly-typed and smoothes out the rough bits of JavaScript."
-"Go",trial,"Languages and Frameworks",true,"<a href='https://golang.org/'>Go</a> is the natural language to use when writing cloud native software."
+"Go",hold,"Languages and Frameworks",true,"<a href='https://golang.org/'>Go</a> is the natural language to use when writing cloud native software. But it is now possible to achieve good results in C# anyway"
```
Comment (clivetong) [inline radar.csv:4]:
```suggestion
"Go",hold,"Languages and Frameworks",true,"<a href='https://golang.org/'>Go</a> is the natural language to use when writing cloud native software. But it is now possible to achieve good results in C# anyway."
```

**Comment by piers-williams** [comment]
Do we need a certain number of reviewers to merge?

**Comment by martin-podlubny** [comment]
> Had a quick natter over coffee with one of Redgate's local Go aficionados, who agrees putting it into hold makes sense given the company's technical proficiencies and the fact that Go is no longer the only option. If that counts as a review.

It was me ✋ Yep, what I was saying that introducing and entertaining new tech should be done with a good reason if there's a cost to it.

The good reason long long time ago, when Foundry/Sharks wrote Spawn (rgclone's predecessor) using golang was that at the time it was pretty much the only sensible way (defo the main community-encouraged) way to write Kubernetes operators. And yeah like the `radar.csv` says, at the time it was pretty much the _"natural language to use when writing cloud native software"_.

The main cost to golang I see in Redgate now is everything you think of when you imagine a building full of C# devs being told to now write golang. Doable, but not free in terms of effort. 

Golang _might_ still be the better tool for the job (compared to C#) in the context of thin cloud-native services and all things Kubernetes, but if the `trial` period of golang on the tech radar showed evidence that it's a headache, and we/Redgate wanna reduce how many languages we're dealing with, then yep, I agree putting golang on hold makes sense :)

**Comment by TheEadie** [comment]
We discussed this in the weekly Architect meeting this week and agree that we should move this to "Hold"

**Pull request merged**